Actions/RunPipeline/RunPipeline.ps1

```diff
@@ -116,8 +124,28 @@ try {
         }
     }
 
-    $repo = AnalyzeRepo -settings $settings -token $token -baseFolder $baseFolder -project $project -insiderSasToken $insiderSasToken @analyzeRepoParams
-    if ((-not $repo.appFolders) -and (-not $repo.testFolders)) {
+    $repo = AnalyzeRepo -settings $settings -token $token -baseFolder $baseFolder -project $project -insiderSasToken $insiderSasToken -doNotCheckAppDependencyProbingPaths @analyzeRepoParams
+
+    if ($appFolders) {
```

Comment:
Absolutely no idea why all this is needed.

So, there's `$appFolders` parameter , there's `$settings.appFolders` and `$repo.appFolders`. At least one isn't needed. Ideally, all of them should be the same, shouldn't they? If not, why?

Reply:
appFolders is the parameter for the action, which is mandatory false.
it defaults to repo.appFolders.
I will make it mandatory = true to simplify this